scapy/arch/windows/__init__.py

```diff
@@ -609,6 +658,11 @@ def read_routes():
     routes = []
     if not conf.prog.os_access:
         return routes
+    global _CACHE_ROUTES
+    if _CACHE_ROUTES:
+        _r = _CACHE_ROUTES
+        _CACHE_ROUTES = None
```

Comment:
Why the code removes the cache on read_routes() call? Could you add a comment if this is needed?

Reply:
We want `read_routes()` to use cached routes only while loading, not when called manually. The cache is loaded once and removed once used.